charts/coturn/templates/secret.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: coturn
+  labels:
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+type: Opaque
+data:
+  zrest_secret.txt: {{ .Values.secrets.zrestSecret | b64enc | quote }}
```

Comment:
Are you sure you wish to base64encode the secret one more time? Is something decoding that secret somewhere?

Reply:
The restund and brig charts both perform similar base64 encoding. This does eventually get decoded somewhere, as in my test environment I've been using during development, the content of the `zrest_secret.txt` file live on the cluster matches the value in the configuration file I feed into Helm. Is it possible that Kubernetes expects file content for Secrets to be base64-encoded when specified like this?